package.json

```
@@ -31,7 +31,7 @@
 		"insert-text-textarea": "^1.0.3",
 		"intervalometer": "^1.0.5",
 		"linkify-issues": "^1.3.0",
-		"linkify-urls": "^2.2.0",
+		"linkify-urls": "^3.1.0",
```

This breaks Firefox, needs to be restored:

```suggestion
		"linkify-urls": "^2.2.0",
```